gammapy/utils/serialization/tests/data/fermi_model.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<source_library title="source library">
+    <source name="3C 273" type="PointSource">
+        <spectrum type="PowerLaw">
+            <parameter free="1" max="1000.0" min="0.001" name="Prefactor" scale="1e-09" value="10"></parameter>
+            <parameter free="1" max="-1.0" min="-5.0" name="Index" scale="1.0" value="-2.1"></parameter>
+            <parameter free="0" max="2000.0" min="30.0" name="Scale" scale="1.0" value="100.0"></parameter>
+        </spectrum>
+        <spatialModel type="SkyDirFunction">
+            <parameter free="0" max="360" min="-360" name="RA" scale="1.0" value="187.25"></parameter>
+            <parameter free="0" max="90" min="-90" name="DEC" scale="1.0" value="2.17"></parameter>
+        </spatialModel>
+    </source>
+    <source name="3C 279" type="PointSource">
+        <spectrum type="PowerLaw">
+            <parameter free="1" max="1000.0" min="0.001" name="Prefactor" scale="1e-09" value="10"></parameter>
+            <parameter free="1" max="-1.0" min="-5.0" name="Index" scale="1.0" value="-2"></parameter>
+            <parameter free="0" max="2000.0" min="30.0" name="Scale" scale="1.0" value="100.0"></parameter>
+        </spectrum>
+        <spatialModel type="SkyDirFunction">
+            <parameter free="0" max="360" min="-360" name="RA" scale="1.0" value="193.98"></parameter>
+            <parameter free="0" max="90" min="-90" name="DEC" scale="1.0" value="-5.82"></parameter>
+        </spatialModel>
+    </source>
+    <source name="gll_iem_v02" type="DiffuseSource">
+        <spectrum type="PowerLaw">
+            <parameter free="1" max="10" min="0" name="Prefactor" scale="1" value="1"></parameter>
+            <parameter free="0" max="1" min="-1" name="Index" scale="1.0" value="0"></parameter>
+            <parameter free="0" max="2e2" min="5e1" name="Scale" scale="1.0" value="1e2"></parameter>
+        </spectrum>
+	<spatialModel file="$GAMMAPY_EXTRA/datasets/fermi_iem/gll_iem_v02.fit" type="MapCubeFunction">
```

Comment:
I didn't push anything to ``gammapy-extra`` yet. This old version of the diffuse model is ~30 MB. OK to add? Or do you have a better idea for a test case?

Reply:
There is e.g. `gammapy-extra/test_datasets/unbundled/fermi/gll_iem_v02_cutout.fits`.
Let's not add a 30 MB file.
IMO even better would be to not use data files if not really needed, and instead create the file in a temp folder in the text with a few lines.
With `gammapy.maps` it's now easy to make any kind of map; for example:
https://github.com/gammapy/gammapy/blob/master/gammapy/cube/tests/test_models.py#L14

Reply:
:+1: